### PR TITLE
fixes #6980

### LINKED ISSUE #6980: In Coppa Field, you can be 999 year old
In Coppa field during registration, you can be 999 year old.

Very small bug but if we're going to ask an age, we should probably make sure it's a valid age.

### PR DESCRIPTION
**Pull request by farhan787**

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fxa-content-server",
-  "version": "1.130.1",
```
Comment (shane-tomlinson) [inline npm-shrinkwrap.json]: Can you revert any changes to the npm-shrinkwrap.json file, this file should only be updated whenever new dependencies are installed or dependencies are updated. `npm install` will cause this to be updated automatically whereas `npm ci` will not.

File: app/scripts/views/elements/coppa-age-input.js [hunk 2]
```diff
@@ -19,6 +19,11 @@ define(function (require, exports, module) {
       const isRequired = typeof this.attr('required') !== 'undefined';
       const value = this.val();
 
+      const maxValidAge = 130;
```
Comment (shane-tomlinson) [inline app/scripts/views/elements/coppa-age-input.js]: Can you extract this a constant to [just below AuthErrors](https://github.com/mozilla/fxa-content-server/pull/6998/files#diff-9617e8f25f3a7d10987ff2be9db46672R11) so that it's easy to find if we ever want to make updates?

File: app/tests/spec/views/elements/coppa-age-input.js [hunk 3]
```diff
@@ -61,6 +61,12 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('invalid', () => {
+        it('if age is greater than 130, throws a `INVALID_AGE`', () => {
```
Comment (shane-tomlinson) [inline app/tests/spec/views/elements/coppa-age-input.js]: Can you also add a test that ensure the boundary condition succeeds, i.e., that `130` does not throw an error?

**Comment by farhan787** [comment]
@shane-tomlinson thanks for your review, I am making a new branch for this as yo mentioned and submitting a new PR to fix this issue [link to new PR](https://github.com/mozilla/fxa-content-server/pull/7001). I think this will be good now 😊.

**Comment by shane-tomlinson** [comment]
Closing this in favor of #7001.
